src/plugins/recordTypes/organization/fields.js

```diff
@@ -0,0 +1,172 @@
+import { defineMessages } from 'react-intl';
+
+export default (pluginContext) => {
+  const {
+    TextInput,
+    OptionPickerInput,
+    AutocompleteInput,
+    TermPickerInput,
+    CompoundInput,
+  } = pluginContext.inputComponents;
+
+
+  const {
+    configKey: config,
+  } = pluginContext.configHelpers;
+
+  return {
+    document: {
+      'ns2:organizations_common': {
+        foundingPlace: {
+          [config]: {
+            view: {
+              type: TermPickerInput,
+              props: {
+                source: 'country',
+              },
+            },
+          },
+        },
+        orgTermGroupList: {
+          orgTermGroup: {
+            termStatus: {
+              [config]: {
+                defaultValue: 'provisional',
```

Comment:
Leave out defaultValue here. In general, only set defaultValue if a default was explicitly set in the 4.1 app layer config. In 4.1 this was just inheriting core behavior, which has changed in 5.0. Let's stick with the default 5.0 behavior. If someone complains, we can add this back.